zh_CN/README.md

```diff
@@ -381,6 +383,8 @@ _可以看到，以上标准均不是绝对条件。我们会进行权衡，并
 
 - [Beego](http://beego.me)：Beego是一个快速开发 Go 应用的 HTTP 框架，他可以用来快速开发 API、Web 及后端服务等各种应用，框架具有简单化、智能化、模块化、高性能等特性。
 - [Macaron](http://go-macaron.com)：高生产力和模块化设计的 Go Web 框架。
+- [Faygo](https://github.com/henrylee2cn/faygo)：Faygo 是一款快速、简洁的Go Web框架，可用极少的代码开发出高性能的Web应用程序（尤其是API接口）。只需定义 struct Handler，Faygo 就能自动绑定、验证请求参数并生成在线API文档。
+- [Teleport](https://github.com/henrylee2cn/teleport)：Teleport是一个通用、高效、灵活的TCP Socket框架。可用于Peer-Peer对等通信、RPC、长连接网关、微服务、推送服务，游戏服务等领域。
```

Comment:
Please place Teleport in class 'Networking'.

Reply:
ok